plugins/woocommerce/client/legacy/css/twenty-twenty-three.scss

```diff
@@ -365,63 +365,6 @@
 
 	}
 
-	// Description/Additional info/Reviews tabs.
-	.woocommerce-tabs {
-		padding-top: var(--wp--style--block-gap);
-
-		ul.wc-tabs {
```

Comment:
Because the block is not inside a `.woocommerce` wrapper in the editor, we needed to move these styles out of `.woocomerce { ... }`. The change shouldn't be problematic because these classes are all prefixed with `.woocommerce-` or `.wc-`, so there won't be conflicts with styles from WP core or other plugins.